.github/workflows/ci.yml

```diff
@@ -17,7 +18,7 @@ env:
 
 jobs:
   engine-tests:
-    runs-on: macos-13
+    runs-on: ubuntu-latest
```

Comment:
Does the Ubuntu image include an emulator? The macOS image was used to allow us to run instrumentation tests.

Reply:
The performance tests require an emulator to run.

Reply:
looks like yes, https://github.com/marketplace/actions/android-emulator-runner

Reply:
Yeah and its actually much faster. Preliminary tests show around 50% improvement for the jobs